Firmware/Source/grovepi/extra_script.py

```diff
@@ -0,0 +1,21 @@
+# Import('env')
```

Comment:
Could you document this one a bit? With a name like "extra_script" and no info inside, it's hard to know what it's for.